homeassistant/components/juicenet/sensor.py

```diff
@@ -13,13 +18,23 @@
 from .entity import JuiceNetDevice
 
 SENSOR_TYPES = {
-    "status": ["Charging Status", None, None],
-    "temperature": ["Temperature", TEMP_CELSIUS, STATE_CLASS_MEASUREMENT],
-    "voltage": ["Voltage", VOLT, None],
-    "amps": ["Amps", ELECTRICAL_CURRENT_AMPERE, STATE_CLASS_MEASUREMENT],
-    "watts": ["Watts", POWER_WATT, STATE_CLASS_MEASUREMENT],
-    "charge_time": ["Charge time", TIME_SECONDS, None],
-    "energy_added": ["Energy added", ENERGY_WATT_HOUR, None],
+    "status": ["Charging Status", None, None, None],
+    "temperature": [
+        "Temperature",
+        TEMP_CELSIUS,
+        DEVICE_CLASS_TEMPERATURE,
```

Comment:
The devices classes added to the sensor types, aren't used by the integration at this point. Forgot to add a line to the entity code?

Reply:
Fixed in 8949ff4